shepherd

```diff
@@ -53,39 +63,44 @@ update_services() {
 }
 
 main() {
-  local blacklist sleep_time supports_detach_option supports_registry_auth
+  local blacklist sleep_time supports_detach_option supports_registry_auth tz verbose
   blacklist="${BLACKLIST_SERVICES:-}"
   sleep_time="${SLEEP_TIME:-5m}"
+  tz="${TZ:-EST}"
+  verbose="${VERBOSE:-true}"
+  
+  echo $tz > /etc/timezone
+  logger "Timezone set to $tz"
```

Comment:
Are you sure this is needed? As the `TZ` variable is set globally for the container AFAIK there is no need to write to the timezone file.